include/openssl/core_dispatch.h

```diff
@@ -97,6 +97,10 @@ OSSL_CORE_MAKE_FUNC(int, core_pop_error_to_mark, (const OSSL_CORE_HANDLE *prov))
 #define OSSL_FUNC_CORE_OBJ_ADD_SIGID          11
 #define OSSL_FUNC_CORE_OBJ_CREATE             12
 
+#define OSSL_FUNC_CORE_CLEAR_LAST_CONSTANT_TIME 13
+OSSL_CORE_MAKE_FUNC(void, core_clear_last_constant_time,
+                    (const OSSL_CORE_HANDLE *prov, int clear))
+
```

Comment:
This creates an incompatibility of a provider using this new core function with older libcrypto. Not sure if this is a problem.